packages/front-end/components/Experiment/ResultsTableTooltip/ResultsTableTooltip.tsx

```diff
@@ -411,103 +413,106 @@ export default function ResultsTableTooltip({
-                {ciRangeText}
-              </div>
-            </div>
+                <div
+                  className={clsx(
+                    "results-ci d-flex mt-1",
+                    data.rowResults.resultsStatus
+                  )}
+                >
+                  <div className="label mr-2">
+                    {data.statsEngine === "bayesian"
+                      ? "95% Credible Interval:"
+                      : `${confidencePct} Confidence Interval:`}
+                  </div>
+                  <div
+                    className={clsx("value nowrap", {
+                      "font-weight-bold": !data.isGuardrail
+                        ? data.rowResults.significant
+                        : data.rowResults.significantUnadjusted,
+                      opacity50: !data.rowResults.enoughData,
+                    })}
+                  >
+                    {ciRangeText}
+                  </div>
+                </div>
 
-            <div
-              className={clsx(
-                "results-chance d-flex mt-1",
-                data.rowResults.resultsStatus
-              )}
-            >
-              <div className="label mr-2">
-                {data.statsEngine === "bayesian"
-                  ? "Chance to Win:"
-                  : "P-Value:"}
-              </div>
-              <div
-                className={clsx("value", {
-                  "font-weight-bold": !data.isGuardrail
-                    ? data.rowResults.significant
-                    : data.rowResults.significantUnadjusted,
-                  opacity50: !data.rowResults.enoughData,
-                })}
-              >
-                {data.statsEngine === "bayesian"
-                  ? percentFormatter.format(data.stats.chanceToWin ?? 0)
-                  : pValText}
-              </div>
-            </div>
-            {addLiftWarning ? (
+                <div
+                  className={clsx(
+                    "results-chance d-flex mt-1",
+                    data.rowResults.resultsStatus
+                  )}
+                >
+                  <div className="label mr-2">
+                    {data.statsEngine === "bayesian"
+                      ? "Chance to Win:"
+                      : "P-Value:"}
+                  </div>
+                  <div
+                    className={clsx("value", {
+                      "font-weight-bold": !data.isGuardrail
+                        ? data.rowResults.significant
+                        : data.rowResults.significantUnadjusted,
+                      opacity50: !data.rowResults.enoughData,
+                    })}
+                  >
+                    {data.statsEngine === "bayesian"
+                      ? percentFormatter.format(data.stats.chanceToWin ?? 0)
+                      : pValText}
+                  </div>
+                </div>
+              </>
+            ) : null}
+            {!addLiftWarning && data.rowResults.enoughData ? (
```

Comment:
question: Why are we inverting the condition here? From should add lift warning to not?

Reply:
Thank you for catching that - this is an artifact from me forcing a variation to display for testing. I'll remove the bang operator.

Reply:
This is what was causing the "CUPER affects results" banner to display in the `after` screenshots.